package.json

```diff
@@ -0,0 +1,50 @@
+{
+	"name": "wikibase-release-pipeline",
+	"version": "3.0.0",
+	"license": "MIT",
```

Comment:
we are actually using BSD 3-Clause https://github.com/wmde/wikibase-release-pipeline/blob/main/LICENSE

Reply:
Updated